src/public_data/index.md

```diff
@@ -0,0 +1,91 @@
+# Public Data
```

Comment:
This section is also targeted towards users outside of Mozilla that want to access the data.